clients/cli/devices.c

```diff
@@ -841,8 +841,8 @@ usage_device_wifi (void)
 	              "ARGUMENTS := [list [ifname <ifname>] [bssid <BSSID>] [--rescan yes|no|auto]]\n"
 	              "\n"
 	              "List available Wi-Fi access points. The 'ifname' and 'bssid' options can be\n"
-	              "used to list APs for a particular interface, or with a specific BSSID. The"
-	              "--rescan flags tells whether a new wifi scan should be triggered.\n"
+	              "used to list APs for a particular interface, or with a specific BSSID. The\n"
+	              "--rescan flag tells whether a new wifi scan should be triggered.\n"
```

Comment:
> cli: fix a couple of typoes 

is the spelling "typoes" here intentional (a.k.a. a joke?).

The spelling fixes is in user-visible + translatable strings. The commit message should point that out, because it makes a difference that it is not a trivial spelling fix in a code comment (which would be far less visible to the user).

Reply:
> is the spelling "typoes" here intentional (a.k.a. a joke?).

No, I honestly thought it's spelled this way.

Will fix that and also enhance the commit message as you suggest.

Reply:
commit message seems not enhanced (yet?).

Should we also refrain from "wifi" and write "Wi-Fi" instead? Above: "... tells whether a new wifi scan ..."

Reply:
> commit message seems not enhanced (yet?).

It is. It says in details what the patch does. It does make no claim that it's trivial, and a negative claim "it's not trivial" doesn't seem appropriate. Most commits are not -- we mark those which positively are.

> Should we also refrain from "wifi" and write "Wi-Fi" instead? Above: "... tells whether a new wifi scan ..."

Yes.